apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -118,72 +192,134 @@ export default class ExportHTMLModal extends Vue {
+    circleNone: this.iconDataToSVG(mdiCircle, 'rgb(3, 169, 244)'),
+    // Low severity; yellow
+    circleLow: this.iconDataToSVG(mdiCircle, 'rgb(255, 235, 59)'),
+    // Medium severity; orange
+    circleMedium: this.iconDataToSVG(mdiCircle, 'rgb(255, 152, 0)'),
+    // High severity; deep orange
+    circleHigh: this.iconDataToSVG(mdiCircle, 'rgb(255, 87, 34)'),
+    // Critical severity; red
+    circleCritical: this.iconDataToSVG(mdiCircle, 'rgb(244, 67, 54)'),
+    // Generic circle
+    circleWhite: this.iconDataToSVG(mdiCircle, 'rgb(0, 0, 0)')
+  };
+
+  // Default attributes
   showingModal = false;
-  exportType = 'executive';
-  description = 'Profile Info\nStatuses\nCompliance Donuts';
+  exportType = FileExportTypes.executive;
+  description = FileExportDescriptions.executive;
+  printHelp = false;
   outputData: OutputData = {
-    bootstrapCSS: '',
-    bootstrapJS: '',
-    jquery: '',
+    tailwindStyles: '',
+    tailwindElements: '',
+    // Used for profile status reporting
     statistics: {
       passed: 0,
       failed: 0,
       notApplicable: 0,
       notReviewed: 0,
+      profileError: 0,
+      totalControls: 0,
       passedTests: 0,
       passingTestsFailedControl: 0,
       failedTests: 0,
       totalTests: 0
     },
+    // Used for profile severity reporting
+    severity: {
+      none: 0,
+      low: 0,
+      medium: 0,
+      high: 0,
+      critical: 0
+    },
+    // Used for profile compliance reporting
+    compliance: {
+      level: '',
+      color: ''
+    },
     files: [],
     controlSets: [],
     showControlSets: false,
     showCode: false,
     exportType: '',
-    complianceCards: [],
+    // Series of icons used for profile-related detail reports
     icons: {
-      circleCheck: this.iconDatatoSVG(mdiCheckCircle, 'white'),
-      circleCross: this.iconDatatoSVG(mdiCloseCircle, 'white'),
-      circleMinus: this.iconDatatoSVG(mdiMinusCircle, 'white'),
-      circleAlert: this.iconDatatoSVG(mdiAlertCircle, 'white')
+      // Passed
+      circleCheck: this.iconHTMLStore.circleCheck,
+      // Failed
+      circleCross: this.iconHTMLStore.circleCross,
+      // Not applicable
+      circleMinus: this.iconHTMLStore.circleMinus,
+      // Not reviewed
+      circleAlert: this.iconHTMLStore.circleAlert,
+      // Profile error
+      triangleAlert: this.iconHTMLStore.triangleAlert,
+      // Total count
+      squareEqual: this.iconHTMLStore.squareEqual,
+      // No severity
+      circleNone: this.iconHTMLStore.circleNone,
+      // Low severity
+      circleLow: this.iconHTMLStore.circleLow,
+      // Medium severity
+      circleMedium: this.iconHTMLStore.circleMedium,
+      // High severity
+      circleHigh: this.iconHTMLStore.circleHigh,
+      // Critical severity
+      circleCritical: this.iconHTMLStore.circleCritical
     }
   };
 
+  // Configures outputData object's report type based on user input
   @Watch('exportType')
   onFileChanged(newValue: string, _oldValue: string) {
```

Comment:
i don't think files are actually changing.  this function should probably be renamed to something more accurately reflecting what's going on.